clients/go/test/api_jobs_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Phrase Strings API Reference
+
+Testing JobsApiService
+
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech);
+
+package phrase
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/phrase/phrase-go/v2"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_phrase_JobsApiService(t *testing.T) {
+	t.Run("Test JobsApiService JobCreate", func(t *testing.T) {
+		server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			// assert request body
+			var requestBody map[string]interface{}
+			decoder := json.NewDecoder(r.Body)
+			if err := decoder.Decode(&requestBody); err != nil {
+				t.Fatal(err)
+			}
+
+			// Access the values from the JSON payload
+			assert.Equal(t, requestBody["name"], "createjobactionb1")
+			assert.Equal(t, requestBody["due_date"], nil)
```

Comment:
I presume the test fails when the change in `/path/jobs` is reverted?

Reply:
Yes, let me add some screenshot to description to make it clear. :)

Reply:
@jablan when I remove nullable from `create.yaml`

![Screenshot 2024-02-05 at 09 43 02](https://github.com/phrase/openapi/assets/141136168/8c2921e2-ad26-4a74-823e-b8b284ffa914)

Reply:
thanks, I wasn't really doubting :)